src/libsystemd-network/sd-dhcp-lease.c

```diff
@@ -1234,8 +1234,7 @@ int dhcp_lease_load(sd_dhcp_lease **ret, const char *lease_file) {
                         return -ENOMEM;
 
                 if (!strv_isempty(a)) {
-                        lease->search_domains = a;
-                        a = NULL;
+                        lease->search_domains = TAKE_PTR(a);
                 }
```

Comment:
{} remains?

Reply:
I see. I extended the `zz-drop-braces` transformation to drop the braces even in cases the single-line statement is an expression instead of a function call. To make this change easier to review, I added it as a separate commit, PTAL.